### PR TITLE
feat: Persistant Task

### PR DESCRIPTION
**Pull request by itsHenry35**
closes https://github.com/alist-org/alist/issues/5313

### CODE REVIEW

**Comment by xhofe** [comment]
Making the feature optional would prevent any issues with it from impacting the software's regular usage in my opinion.

**Comment by itsHenry35** [comment]
@xhofe I've added the switch, please re-review this PR

File: internal/bootstrap/data/setting.go [hunk 1]
```diff
@@ -144,6 +144,7 @@ func InitialSettings() []model.SettingItem {
 		{Key: conf.ForwardDirectLinkParams, Value: "false", Type: conf.TypeBool, Group: model.GLOBAL},
 		{Key: conf.IgnoreDirectLinkParams, Value: "sign,alist_ts", Type: conf.TypeString, Group: model.GLOBAL},
 		{Key: conf.WebauthnLoginEnabled, Value: "false", Type: conf.TypeBool, Group: model.GLOBAL, Flag: model.PUBLIC},
+		{Key: conf.IsTaskPersistant, Value: "true", Type: conf.TypeBool, Group: model.GLOBAL},
```
Comment (xhofe) [inline internal/bootstrap/data/setting.go:147]: I suggest placing this switch within the configuration file, as modifying it directly in the database can be challenging if this feature disrupts the program's normal operation.

Comment (xhofe) [reply internal/bootstrap/data/setting.go:147]: Additionally, the configuration file contains distinct settings for each task, allowing for individual customization of each one.

Comment (itsHenry35) [reply internal/bootstrap/data/setting.go:147]: Got it, I will update the code.

**Comment by itsHenry35** [comment]
@xhofe I've committed the change, pls review it again.

**Review by xhofe — APPROVED** [review]

**Pull request merged**